edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss

```diff
@@ -1,46 +1,44 @@
-div.login-register-content {
-  background: $background-color-form-login-register;
-}
-
-.login-register .login-provider {
-  width: auto !important;
-  padding-right: 10px !important;
-  min-width: 165px;
-}
-
-.login-register {
-  background: $background-color-form-login-register;
-}
-
-form.login-form {
-  button.action.action-primary.action-update.js-login.login-button {
-    background: $primary;
-    color: $inverse;
-  }
-
-  button.action.action-primary.action-update.js-register.register-button {
-    background: $primary;
-    color: $inverse;
-  }
-}
-
-.login-register .form-field select {
-  background: $inverse;
-}
-
-
-button.action.action-primary.action-update.js-reset {
-  background: $primary;
-}
-
-
-.login-register .status .message-copy {
-  p, p a, li {
-    color: $secondary;
-  }
-}
-
-
-div.login-register div.toggle-form a.form-toggle {
-  color: $primary;
+.signin-container {
+    box-shadow: 2px 2px 2px #eee;
+    padding: 50px 30px 60px;
+}
+.signing-title {
+    text-align: center;
+    font-size: 18px;
+    font-weight: 600;
+}
+.signing-title span {
+    margin-bottom: 10px;
+}
+.login-register .form-wrapper .login-providers {
+    display: flex;
+    text-align: left;
+    background: #f8f8f8;
```

Comment:
Can you get this color from variables definitions? @luismorenolopera

Reply:
yes